### ISSUE TITLE
More detailed method description in recipes

### ISSUE THREAD

**Issue by kehanlu** [speechbrain/speechbrain#961]
Hi,
I found in many recipes, the descriptions did not reveal full methods they used in the recipe. Make it difficult to make reasonable comparison with other toolkits or methods in a glance.

For example, in `recipes/AISHELL-1/ASR/transformer`, the reported results in both recipes used spec augmentation and rirs noise augmentation to improve the error rates, but did not specify in README.md. Anyone who wants to know what they really do have to read the code and hparams file by themselves. 

I suggest that we can design some guide for contributors to report their recipes. For example paper they implemented, augmentation method they used or other detailed config such as kernel size, training epochs etc.

**Comment by TParcollet** [comment]
@mravanelli what do you think ?

**Comment by mravanelli** [comment]
Yes, I think we can provide more details in the README file if the users
think that this is necessary.

**Comment by Adel-Moumen** [comment]
Hey, 

I think we could create a template for README.md with a section describing the systems.. what do you think? 

Best.

**Comment by Noel6161131110** [comment]
Hey there,

should we edit on the existing readme..😀
